marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -302,10 +308,10 @@ groupTransactionInputByParticipant (TransactionInput { inputs, interval }) =
       (NEA.head nea # \{ party } -> { inputs: [], party, interval })
```

Comment:
A tiny thing, and not related to this PR - but I prefer `NonEmptyArray` for clarity.

Reply:
👍🏻